cmd/main.go

```diff
@@ -156,6 +159,23 @@ func main() {
 	connectivityValidator := connectivity.NewValidator(log.WithField("pkg", "validators"))
 	instructionApi := host.NewInstructionManager(log.WithField("pkg", "instructions"), db, hwValidator, Options.InstructionConfig, connectivityValidator)
 
+	var imagesWithAuth = []string{
```

Comment:
if it's a default configuration then it's belongs to the package.

Reply:
I don't think it can be in a package because it references images from different subsystems. Only at the top level we know which registries are being used across the application.